delphi-checks/src/main/java/au/com/integradev/delphi/checks/LowercaseKeywordCheck.java

```diff
@@ -34,7 +34,7 @@
 @DeprecatedRuleKey(ruleKey = "LowerCaseReservedWordsRule", repositoryKey = "delph")
 @Rule(key = "LowercaseKeyword")
 public class LowercaseKeywordCheck extends DelphiCheck {
-  private static final String MESSAGE = "Lowercase this keyword.";
+  private static final String MESSAGE = "Lowercase this keyword (found: \"%s\" expected: \"%s\").";
```

Comment:
Can you inline `MESSAGE` into the usage in `withMessage` to fix the failing build?
ErrorProne doesn't like it when format-strings with 1 usage are pulled out into constants.